### PR TITLE
Try to set required ini values in base.php

### LINKED ISSUE #16006: PHP settings for installation
Installing ownCloud locally, I got prompted with this:
screenshot from 2015-05-01 22 17 40

Default ubuntu 14.04 
PHP 5.5.9-1ubuntu4.9 (cli) (built: Apr 17 2015 11:44:57)
Latest git master

### PR DESCRIPTION
**Pull request by LukasReschke**
Fixes https://github.com/owncloud/core/issues/16006

cc @karlitschek @DeepDiver1975

### CODE REVIEW

**Comment by LukasReschke** [comment]
We now have:

1. The value defined in `.htaccess` and `.user.ini`
2. This `ini_set` in base.php

I guess that should be enough for properly configured PHP hosts. If somebody has neither of both allowed then manually changing should be legitimate.

File: lib/base.php [hunk 1]
```diff
@@ -559,6 +565,7 @@ public static function init() {
 		@ini_set('post_max_size', '10G');
 		@ini_set('file_uploads', '50');
```
Comment (MorrisJobke) [inline lib/base.php]: Move the 6 set statements above also to this method?

Comment (LukasReschke) [inline lib/base.php]: I'd want to avoid this. Those are not required while the one in the function are pretty much required to even run ownCloud.

Comment (MorrisJobke) [inline lib/base.php]: But then has the potential to get moved in there later. Maybe add a method: `setOptionalIniValues()`?

**Comment by karlitschek** [comment]
tested and works. Set the default charset in php.ini to something other the utf-8 -> setup still works :+1:

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**